docker/luna_tutorial/Makefile

```diff
@@ -46,27 +46,36 @@ help:
 
 
 clean:      ## cleanup luna_tutorial images and containers
-	docker rmi $$(docker images luna* -q)
-	docker rmi $$(docker container ls -f "name=luna*" -q)
+	-docker rmi $$(docker images luna* -q)
+	-docker rmi $$(docker container ls -f "name=luna*" -q)
 
 
 clean-image:  ## remove luna tutorial image
-	docker rmi -f $$(docker images luna* -q)
+	-docker rmi -f $$(docker images luna* -q)
 
 
 build:        ## build containers
 	mkdir -p vmount/logs vmount/db vmount/tmp
-	USER_ID=$(USER_ID) USER_NAME=$(USER_NAME) GROUP_NAME=$(GROUP_NAME) GROUP_ID=$(GROUP_ID) PWD=$(PWD) DOCKER_PATH=$(DOCKER_PATH) HOST_IP=$(HOST_IP) HOST_PORT=$(HOST_PORT) docker-compose --verbose build
-
-
-run:        ## launch containers. Default IP:PORT=127.0.0.1:8888. Optionally specify args HOST_IP=xxx.xxx.xxx.xxx HOST_PORT=xxxx.
-	mkdir -p vmount/logs vmount/db vmount/tmp
-	USER_ID=$(USER_ID) USER_NAME=$(USER_NAME) GROUP_NAME=$(GROUP_NAME) GROUP_ID=$(GROUP_ID) PWD=$(PWD) DOCKER_PATH=$(DOCKER_PATH) HOST_IP=$(HOST_IP) HOST_PORT=$(HOST_PORT) docker-compose --log-level INFO --verbose up -d
-
+	USER_ID=$(USER_ID) USER_NAME=$(USER_NAME) GROUP_NAME=$(GROUP_NAME) GROUP_ID=$(GROUP_ID) PWD=$(PWD) DOCKER_PATH=$(DOCKER_PATH) docker-compose build
+
+
+run:        ## launch containers
+	./apply_ports.sh
+	mkdir -p vmount/logs vmount/db vmount/tmp vmount/assetstore
+	USER_ID=$(USER_ID) USER_NAME=$(USER_NAME) GROUP_NAME=$(GROUP_NAME) GROUP_ID=$(GROUP_ID) PWD=$(PWD) DOCKER_PATH=$(DOCKER_PATH) docker-compose up -d
+	@echo
+	@echo DSA will take a few minutes to become available at,
```

Comment:
"DSA is being set up with sample images, annotations and admin user (admin/password)
It will take a couple minutes to become available at"